content/docs/for-buildpack-authors/how-to/write-buildpacks/clear-env.md

```diff
@@ -3,8 +3,23 @@ title="Clear the buildpack environment"
 weight=99
 +++
 
+During the `detect` and `build` phases, the lifecycle MUST provide as environment variables any user-provided files in `<platform>/env/` with environment variable names and contents matching the file names and contents.
+
 <!--more-->
 
-This page is a stub! The CNB project is applying to [Google Season of Docs](https://developers.google.com/season-of-docs/docs/timeline) to receive support for improving our documentation. Please check back soon.
+The `lifecycle` MUST also provide as environment variables any operator-provided files in `<build-config>/env` with environment variable names and contents matching the file names and contents. This applies for all values of `clear-env` or if `clear-env` is undefined in `buildpack.toml`.
```

Comment:
I think this bit of detail if maybe a little confusing in the context. We can just say something like 

> Note that buildpacks CANNOT declare "I am not configurable by the platform". That is a separate thing.

above (but more professionally 🙃 )

Reply:
added

Reply:
added